plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -209,11 +186,105 @@ class NetworkingFormSection extends mixin(StoreMixin) {
                 </Tooltip>
               </FormGroupHeadingContent>
             </FormGroupHeading>
-            <FieldHelp>
-              Load balance this service internally at {hostName}
-            </FieldHelp>
           </FieldLabel>
-          <FieldError>{vipPortError || loadBalancedError}</FieldError>
+        </FormGroup>
+      </FormRow>,
+      (loadBalanced && this.getLoadBalancedPortField(endpoint, index))
+    ];
+  }
+
+  getLoadBalancedPortField(endpoint, index) {
+    const {errors, data: {id, portsAutoAssign}} = this.props;
```

Comment:
It won't as we have it in the `defaultProps` that will also prevent destruction error.